index.html

```diff
@@ -11,13 +11,17 @@
 </head>
 <body>
     <header>
-        <a href="index.html" class="logo"><img src="assets/images/Logo Placeholder.png" alt="Portfolio Logo"></a>
+        <a id="logo" href="index.html"><img src="assets/images/Logo Placeholder.png" alt="Portfolio Logo"></a>
         <nav>
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you add a position of sticky to your Navbar to make it always visible to the user even when they click on any of the mobile menu options.